### DISCUSSION TITLE
APRS IGate Master, Mehr Wlan eintragungen

### DISCUSSION THREAD

**Discussion by Do1SKL** [lora-aprs/LoRa_APRS_iGate#209] [Ideas]
Hallo

Wäre es nicht möglich das man mehrere Wlanzugänge erstellen kann ? z.b Wlan:1 Zuhause, WLAN:2 Handy hotspot  usw ? man wäre flexiebler ,da man nicht immer neu schreiben müsste , so würde sich das igate den zugang nehmen der verfügbar ist .

Gruß Sebastian 
DO1SKL

**Comment by Mane76** [comment]
Hallo DO1SKL, kannst Du in der is-cfg.json in der wifi-section:

    "wifi": {
        "active": true,
        "AP": [
            {
                "SSID": "erstes WLAN",
                "password": "erstes Passwort"
            },
            {
                "SSID": "zweites WLAN",
                "password": "zweites Passwort"
            },
            {
                "SSID": "drittes WLAN",
                "password": "drittes Passwort"
            }
        ]
    },


Grüße
Mane
